filesystems/gcs/src/main/scala/cromwell/filesystems/gcs/GcsPathBuilder.scala

```diff
@@ -53,10 +53,21 @@ class GcsPathBuilder(authMode: GoogleAuthMode,
                      options: WorkflowOptions) extends PathBuilder {
   authMode.validate(options)
 
-  protected val storageOptions = StorageOptions.builder()
-    .authCredentials(authMode.authCredentials(options))
-    .retryParams(retryParams)
-    .build()
+  val googleProjectProvided = options.getOrElse("google_project", None)
+
```

Comment:
I believe what you should be able to do is something like this (I'm just free typing it, may not even compile as-is)

    val foo = StorageOptions.builder()
                     .authCredentials(authMode.authCredentials(options))
                     .retryParams(retryParams)
    val blah = googleProjectProvided match {
      case project: String => foo.projectId(project)
      case _ => foo
    }
    blah.build()

That looks like it could be done better but the point being that you can do most of the builder and then tack on the `projectId` part as necessary.